.github/workflows/render-docs.yml

```
@@ -38,6 +38,14 @@ on:
         description: 'Commit message'
         default: 'Update documentation'
         type: string
+      committer-name:
+        description: 'Username to use as author of the commit.'
+        default: 'github-actions[bot]'
+        type: string
+      committer-email:
+        description: 'Email address to use as author of the commit.'
+        default: '41898282+github-actions[bot]@users.noreply.github.com'
```

```suggestion
        default: 'github-actions[bot]@users.noreply.github.com'
```
I think I'd rather put a generic e-mail as default and set the Arduino specific one in the repositories that use this workflow. Alternatively we could remove the default value and make this a required parameter.

---

...ah, wait, so 41898282 is already a "generic" user ID?

---

It is the user ID for the "github-actions[bot]" user:

https://api.github.com/users/github-actions[bot]

Note this part:

```json
  "created_at": "2018-07-30T09:30:16Z",
```

The "noreply" email format is documented here:

https://docs.github.com/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address

> If you created your account after July 18, 2017, your noreply email address is an ID number and your username in the form of [ID+USERNAME@users.noreply.github.com](mailto:ID+USERNAME@users.noreply.github.com).

Since the "github-actions[bot]" account was created after 2017-07-18, the email address for the  user is `41898282+github-actions[bot]@users.noreply.github.com` (`41898282` is the user ID for github-actions[bot]).